atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -250,8 +250,8 @@ def set_group_manifest_info(self, extra, worker_metadatas):
             index['tags'] = tags
             index['floating_tags'] = floating_tags
             index['unique_tags'] = unique_tags
-            repositories = self.workflow.build_result.annotations['repositories']['unique']
-            repo = ImageName.parse(repositories[0]).to_str(registry=False, tag=False)
+            repositories = self.workflow.builder.base_image
+            repo = ImageName.parse(repositories).to_str(registry=False, tag=False)
```

Comment:
Should we change the variable name as well since it is not a list anymore?
Was it ever a list?